python/doc/examples/probabilistic_modeling/distributions/plot_create_random_mixture.py

```diff
@@ -50,12 +50,12 @@
 print("Mean : %.3f" % mean)
 
 # %%
-# its variance :
+# Its variance :
```

Comment:
There is no space in front of a comma in english (see [here](https://english.stackexchange.com/questions/4645/is-it-ever-correct-to-have-a-space-before-a-question-or-exclamation-mark)).
```suggestion
# Its variance:
```